RobotCode/src/main/java/frc/robot/commands/AimShooterCommand.java

```diff
@@ -0,0 +1,89 @@
+package frc.robot.commands;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.IndexerConstants;
+import frc.robot.constants.ShooterConstants;
+import frc.robot.constants.ShooterMountConstants;
+import frc.robot.subsystems.ShooterMountSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.VisionSubsystem;
+
+public class AimShooterCommand extends Command
+{
+
+	private ShooterSubsystem shooter;
+	private ShooterMountSubsystem shooterMount;
+	private VisionSubsystem vision;
+
+	private Translation2d shooterMountPose, speakerPose;
+
+	private AprilTagFieldLayout aprilTagFieldLayout;
+
+	public AimShooterCommand(ShooterSubsystem shooter, ShooterMountSubsystem shooterMount,
+			VisionSubsystem vision)
+	{
+		this.shooter = shooter;
+		this.shooterMount = shooterMount;
+		this.vision = vision;
+
+		addRequirements(shooter, shooterMount, vision);
+
+		aprilTagFieldLayout = AprilTagFields.k2024Crescendo.loadAprilTagLayoutField();
```

Comment:
Vision loads the field layout. We could just save it there and then reference it here.